walmart/walmart.py

```diff
@@ -121,7 +121,7 @@ def send_request(
                     ))
                 elif response.status_code == 400:
                     data = response.json()
-                    if data["error"][0]["code"] == \
+                    if "error" in data and data["error"][0]["code"] == \
```

Comment:
So not handling the case where errors is sent in response?

Reply:
Can be handled by upstream, this is only handled to re-authenticate